### ISSUE TITLE
macOS: Graphical artifacts obstruct the Viewport on a MacBook Air 2020 machine with M1 chipset

### ISSUE THREAD

**Issue by LB-MateuszSzweda** [o3de/o3de#7675]
**Describe the bug**
All of the Editor's Viewport on MacBook Air 2020 machine with M1 chipset is obstructed by graphical artifacts when any level is opened.

Please note that the Asset Processor does not display any failed assets and the issue did not reproduce on macOS machines without the M1 chipset.

The issue was observed on AutomatedTesting and new projects.

For further information please refer to the attached video file and log files.

**Steps to reproduce**

1. Obtain the Development branch build via GitHub on macOS machine with M1 chipset.
2. Create a new or move your preexisting 3rdParty folder containing CMake into the o3de folder.
3. Obtain python packages with the _get_python.sh_ script located in the _o3de/python_ directory.
4. Create the _osx_build_ folder under the o3de directory.
5. Set the following temporary environment variables in the Terminal:
```
export PATH=$PATH:/Users/<home_folder_name>/o3de/3rdParty/CMake/CMake.app/Contents/bin
export LY_3RDPARTY_PATH=/Users/<home_folder_name>/o3de/3rdParty
export LY_PACKAGE_SERVER_URLS=https://d2c171ws20a1rv.cloudfront.net
```
6. Configure and generate the Xcode solution with the following command:
`cmake .. -G "Xcode" -DLY_3RDPARTY_PATH=$LY_3RDPARTY_PATH -DLY_PROJECTS=AutomatedTesting -DLY_UNITY_BUILD=1 -DCMAKE_OSX_ARCHITECTURES=x86_64`

7. Build Mac binaries with the following command:
`cmake -E time cmake --build . --target ALL_BUILD --config profile`

8. Launch the built AutomatedTesting project and create a new level.

**Expected behavior**
No graphical artifacts are visible in the Editor's Viewport when a level is launched. 

**Actual behavior**
Graphical artifacts obstruct all of the Viewport when a level is launched.

**Screenshots/Video**

https://user-images.githubusercontent.com/86953456/154279187-7938abf1-f574-4d36-8d2f-0c207563abc9.mov

**Found in Branch**
Development (16d6cbc)

**Desktop/Device:**

- Device: Mac
- OS: macOS
- Version: Monterey 12.0.1
- CPU: M1
- GPU: M1
- Memory: 16GB

**Additional context**
[Editor.log](https://github.com/o3de/o3de/files/8080389/Editor.log)
[Asset Processor log.txt](https://github.com/o3de/o3de/files/8080390/Asset.Processor.log.txt)
[build log.txt](https://github.com/o3de/o3de/files/8080391/build.log.txt)

**Comment by superkitcath** [comment]
Adding sig/graphics, since mougglis is currently working on this one. Internal Ref ATOM-17326

**Comment by LB-MateuszSzweda** [comment]
Verified as fixed in Development (3200c2f).
